config/dev1/sys.config

```diff
@@ -19,7 +19,7 @@
       {db_path, "."},
       {persist, false},
       {autostart, false},
-      {aec_pow_cuckoo, {"lean16", "-t 5", 16}}
+      {aec_pow_cuckoo, {"lean28", "-t 5", 28}}
```

Comment:
Should this change really be global, and not just for the test?

Reply:
Tests use lean16 rather than 28 because faster.

Reply:
Thsi change should not be there and is a mistake. I am now rebuilding and testing to see if I can push an update